pkg/compliance/bsi.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 var (
-	validBsiSpdxVersions = []string{"SPDX-2.3"}
```

Comment:
This is incorrect for BSI 1.1 its SPDX-2.3 only.

Reply:
addressed it !!